File: Ejercicios Propuestos 2/Ejercicio 18/Ejercicio 18.jl
println("Ingrese el número del mes (1 al 12):")
mes = parse(Int, readline())

if mes == 1
    println("Enero - 31 días")
elseif mes == 2
    println("Febrero - 28 días")
elseif mes == 3
    println("Marzo - 31 días")
elseif mes == 4
    println("Abril - 30 días")
elseif mes == 5
    println("Mayo - 31 días")
elseif mes == 6
    println("Junio - 30 días")
elseif mes == 7
    println("Julio - 31 días")
elseif mes == 8
    println("Agosto - 31 días")
elseif mes == 9
    println("Septiembre - 30 días")
elseif mes == 10
    println("Octubre - 31 días")
elseif mes == 11
    println("Noviembre - 30 días")
elseif mes == 12
    println("Diciembre - 31 días")
else
    println("Mes inválido")
end

File: Ejercicios Propuestos 2/Ejercicio 18/Ejercicio 18.java
import java.util.Scanner;

public class EJERCICIO18DeGRUPO2 {
    public static void main(String[] args) {
        // Definir mes Como Entero
        try (Scanner sc = new Scanner(System.in)) {
            // Definir mes Como Entero
            int mes;
            
            System.out.println("Ingrese el número del mes (1 al 12):");
            mes = sc.nextInt();
            
            // Segun mes Hacer
            switch (mes) {
                case 1 -> System.out.println("Enero - 31 días");
                // Detiene la ejecución para que no pase al siguiente mes
                case 2 -> System.out.println("Febrero - 28 días");
                
                case 3 -> System.out.println("Marzo - 31 días");
                
                case 4 -> System.out.println("Abril - 30 días");
                
                case 5 -> System.out.println("Mayo - 31 días");
                
                case 6 -> System.out.println("Junio - 30 días");
                
                case 7 -> System.out.println("Julio - 31 días");
                
                case 8 -> System.out.println("Agosto - 31 días");
                
                case 9 -> System.out.println("Septiembre - 30 días");
                
                case 10 -> System.out.println("Octubre - 31 días");
                
                case 11 -> System.out.println("Noviembre - 30 días");
                
                case 12 -> System.out.println("Diciembre - 31 días");
                
                default -> // Reemplaza a "De Otro Modo:"
                    System.out.println("Mes inválido");
            }
        }
    }
}
